frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -340,6 +404,14 @@ To:
             },
         }
     }
+
+    for _, widget in pairs(self.registered_widgets) do
+        local ok, err = pcall(widget.addToMainMenu, widget, self.menu_items)
```

Comment:
Just asking: why this had to be moved from top of the function to there ?

Reply:
To allow Coverbrowser plugin to place its settings submenu to `self.menu_items.filebrowser_settings`.